justfile

```diff
@@ -32,4 +23,15 @@ install_linkerd: && link_clusters
 
 #link created two multiclusters
 link_clusters:
-   cd utils && ./link_multilclusters.sh
\ No newline at end of file
+   cd utils && ./link_multilclusters.sh
+
+export K3D_ORG_DOMAIN = env_var_or_default("K3D_ORG_DOMAIN", "cluster.local")
+export K3D_NETWORK_NAME = env_var_or_default("K3D_NETWORK_NAME")
+export K3D_CLUSTER_NAME = env_var_or_default("K3D_CLUSTER_NAME")
```

Comment:
`just --eval` is useful for evaluating evars. We can provide custom config and defaults through evars, this will allow us to override options without making any changes to the script itself.